### ISSUE TITLE
Variation Price on archive page

### ISSUE THREAD

**Issue by pixsla** [dudaster/ele-custom-skin#244]
Hello,
I'm using a plugin to display all variation products on the archive page. But as soon as I create a loop for the archive page, everything works fine as well as the variation product images but the prices are not displayed.

**Comment by dudaster** [comment]
can you provide an url?

**Comment by pixsla** [comment]
sure, its:
https://x-upgrade.com/produkt-kategorie/tauleinen/

**Comment by dudaster** [comment]
Have you tried adding add to cart button widget?

**Comment by pixsla** [comment]
Yes, tried already many widgets, as well as normal Text Widget with dynamic "Productprice" content, normal Price Widget and couple add to Cart Widgets.

**Comment by lopezdezign** [comment]
I am wondering if this has been resolved. I am having the same issue on a newly designed site with Ele Custom Skins Pro...
